README.md

```diff
@@ -62,6 +62,13 @@ import lomap
 
 db_mol = lomap.DBMolecules("python string pointing to a directory with mol2 files", output=True)
 
+    #More graphing options:
+    # Use the complete radial graph option. The radial center (lead compound) is automatically picked by the structure similarity: the ligand shared the most structural similarity with others will be picked as lead compounds
```

Comment:
Minor grammar stuff here; I can clean it up after merge if needed. It would be something like "... the ligand with the most structural similarity to all of the others will be picked as 
 the 'lead compounds' and used as the central compound..." or something along those lines.